includes/admin/list-tables/class-wc-admin-list-table-orders.php

```diff
@@ -153,7 +153,8 @@ protected function prepare_row_data( $post_id ) {
 		global $the_order;
 
 		if ( empty( $this->object ) || $this->object->get_id() !== $post_id ) {
-			$this->object = $the_order = wc_get_order( $post_id );
+			$this->object = wc_get_order( $post_id );
+			$the_order    = wc_get_order( $post_id );
```

Comment:
You can save a lot of DB/function calls if you do `$the_order = $this->object` instead.

Reply:
Thanks for taking the time to review this PR, @claudiulodro.

I might be missing something but I don't think the code above will generate any extra DB calls and the extra function calls are negligible:

```
wp> global $wpdb
wp> $wpdb->num_queries
=> int(25)
wp> $o = wc_get_order(68)
wp> $wpdb->num_queries
=> int(28)
wp> $o = wc_get_order(68)
wp> $wpdb->num_queries
=> int(28)
```

I considered doing what you suggested but without thinking too much about it, I decided to go with another approach because I didn't want to make `$the_order` a reference to `this->object` as I don't know how and where in the codebase they are used so I didn't want a change in one to impact the other (specially since `$the_order` is a global that can be changed anywhere):

```
wp> $o1 = wc_get_order(68)
wp> $o2 = $o1
wp> $o1->foo
=> string(0) ""
wp> $o2->foo
=> string(0) ""
wp> $o1->foo = 'bar'
=> string(3) "bar"
wp> $o2->foo
=> string(3) "bar"
```

What do you think?

Reply:
Forget what I said above. I failed to consider that the original code already makes the two variables point to the same object, so what I was worried is not really a problem in this case. I'm going to implement the change you suggested and merge this PR.